include/z64actor.h

```diff
@@ -395,6 +402,21 @@ typedef enum {
 // Target size when activated
 #define LENS_MASK_ACTIVE_SIZE 100
 
+#define HALFDAYBIT_DAY0_DAWN  (1 << 9)
+#define HALFDAYBIT_DAY0_NIGHT (1 << 8)
```

Comment:
These aren't limited to dawn right? If I load a scene at 4pm on day 1, it will still look to `HALFDAYBIT_DAY1_DAWN` to determine if it should be spawned or not. I can see why `HALFDAYBIT_DAY0_DAY` might be weird though because of the double day (I blame English for having day mean both the full 24 day and the daytime with sunlight)

Reply:
Nope, it isn't limited to the actual dawn. Here "Dawn" refers to the period of time which goes from 6 AM to 6 PM (So yes, 4 PM day 1 will check `HALFDAYBIT_DAY1_DAWN`). 
I went with "Dawn" and "Night" because that's the way the game calls those time ranges and as you said `DAY0_DAY` would be confusing. Maybe we could try `NIGHTN'T` :laughing:

Reply:
The only other thing I can think of is `HALFDAYBIT_DAY0_DAYTIME`